tools/bio3d/.shed.yml

```diff
@@ -0,0 +1,10 @@
+name: bio3d
```

Comment:
@tsenapathi can you please make this file look like https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/pull/4/files#diff-63060d7c0c68d44883512a5b909d1c17

Then every tool gets its own toolshed repo :)

Reply:
ping @tsenapathi